app/src/main/res/layout/activity_main.xml

```diff
@@ -1,50 +1,59 @@
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/coordinator"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/background_color">
+<layout>
+    <data>
+        <variable
+            name="viewModel"
+            type="com.example.internetshop.presentation.viewModel.TitleViewModel" />
+    </data>
 
-    <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/app_bar"
+    <androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/coordinator"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="@null">
+        android:layout_height="match_parent"
+        android:background="@color/background_color">
 
-        <com.google.android.material.appbar.CollapsingToolbarLayout
-            android:id="@+id/collapsing_layout"
+        <com.google.android.material.appbar.AppBarLayout
+            android:id="@+id/app_bar"
             android:layout_width="match_parent"
-            android:layout_height="@dimen/dimen_collapsing_tool_bar_height"
-            app:collapsedTitleGravity="center"
-            app:collapsedTitleTextAppearance="@style/CollapsingToolbar"
-            app:contentScrim="@color/white"
-            app:expandedTitleGravity="start|bottom"
-            app:expandedTitleMarginStart="16dp"
-            app:expandedTitleTextAppearance="@style/ExpandedToolBar"
-            app:layout_scrollFlags="scroll|exitUntilCollapsed"
-            tools:title="@string/login">
+            android:layout_height="wrap_content"
+            android:background="@null">
 
-            <androidx.appcompat.widget.Toolbar
-                android:id="@+id/toolbar"
+            <com.google.android.material.appbar.CollapsingToolbarLayout
+                android:id="@+id/collapsing_layout"
                 android:layout_width="match_parent"
-                android:layout_height="?attr/actionBarSize"
-                app:contentInsetEnd="?attr/actionBarSize"
-                app:contentInsetLeft="?attr/actionBarSize"
-                app:contentInsetRight="?attr/actionBarSize"
-                app:contentInsetStart="?attr/actionBarSize"
-                app:contentInsetStartWithNavigation="?attr/actionBarSize"
-                app:layout_collapseMode="pin"
+                android:layout_height="@dimen/dimen_collapsing_tool_bar_height"
+                app:collapsedTitleGravity="center"
+                app:collapsedTitleTextAppearance="@style/CollapsingToolbar"
+                app:contentScrim="@color/white"
+                app:expandedTitleGravity="start|bottom"
+                app:expandedTitleMarginStart="16dp"
```

Comment:
You have forgotten about dimen. Please use value from dimens.xml

Reply:
fixed